### PR TITLE
Update user guide

### PR DESCRIPTION
**Pull request by xzynos**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
+* [Understanding the Command Syntax](#understanding-the-command-syntax)
 * [Features](#features)
    * [Managing Your Expenses](#managing-your-expenses)
    * [Managing Your Recurring Payments](#managing-your-recurring-payments)
    * [Managing Your Incomes](#managing-your-incomes)
    * [Managing Your Targets](#managing-your-targets)
    * [Handling Your Data](#handling-your-data)
    * [Listing all available commands: `help`](#listing-all-available-commands-help)
-   * [Exiting the application: `exit`](#exiting-the-application-exit)
+   * [Exiting the application: `exit`](#exiting-the-application-bye)
```
Comment (jeyvia) [inline docs/UserGuide.md:15]: Should it be "Exiting the application: `bye`" ?

Comment (xzynos) [reply docs/UserGuide.md:15]: Thanks for catching that! I will make the change!

**Comment by jeyvia** [comment]
Looks good, will proceed to merge!

**Comment by jeyvia** [comment]
Looks good will proceed to merge!

**Pull request merged**